bedrock/products/templates/products/vpn/windows-download.html

```diff
@@ -0,0 +1,101 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "products/vpn/download.html" %}
+
```

Comment:
Question (potentially blocking): As these pages auto-download a file on load, we probably want to exclude them from search engines? Worth confirming with Adria here?

```
{% block canonical_urls %}<meta name="robots" content="noindex,follow">{% endblock %}
```

Reply:
I'll ask Adria :)

Reply:
She agreed that we should noindex the page, and she also mentioned to nofollow the links to the post-download pages, so I'll be adding `rel="nofollow"` in the `/vpn/download` page